### PR TITLE
Feat/report get endpoint

### PR DESCRIPTION
**Pull request by fahmirzldi**
add get_all_reports and get_report_by_id

### CODE REVIEW

**Comment by fahmirzldi** [comment]
kindly review @fatjan @AchmadQiz

File: app/main/controller/report_controller.py [hunk 1]
```diff
@@ -15,4 +15,25 @@ class ReportList(Resource):
...
-        return create_report(ns.payload, user_id)
\ No newline at end of file
+        return create_report(ns.payload, user_id)
+    
+    @ns.doc(security="bearer")
+    @token_required
+    def get(self, decoded_token):
+        # role = decoded_token["role"]
+        # if role != "admin":
+        #     return error_handler("Access denied")
```
Comment (fatjan) [inline app/main/controller/report_controller.py:25]: why is this commented?

does it mean the user with user role can also a list of report that was sent towards them?

File: tests/model/test_report_model.py [hunk 2]
```diff
@@ -29,6 +29,20 @@ def register_user():
     user_model = User()
     user = user_model.register_user(user_data)
     user_id = user_model.get_user_id(user["public_id"])
+    user_role = user_model.get_user_id(user["role"])
```
Comment (fatjan) [inline tests/model/test_report_model.py:32]: why is the same method used with different argument giving different output?

File: app/main/controller/report_controller.py (hunk 1 above)
Comment (fahmirzldi) [reply app/main/controller/report_controller.py:25]: Oh it should be uncommented

File: tests/model/test_report_model.py (hunk 2 above)
Comment (fahmirzldi) [reply tests/model/test_report_model.py:32]: Oh yeah I just realize its weird, I will add get_user_role method in user model

Comment (fahmirzldi) [reply tests/model/test_report_model.py]: Okay done please kindly review kak, it weird that it didn't throw any error before

Comment (fatjan) [reply tests/model/test_report_model.py]: I checked out to your branch and run flask test, this test failed:
`tests/controller/test_report_controller.py::TestReportEndpoints::test_get_all_reports FAILED  `

You might want to check it further

Comment (fahmirzldi) [reply tests/model/test_report_model.py]: Fixed, kindly review again @fatjan

**Review by fatjan — APPROVED** [review]

**Pull request merged**